frontend/src/App.js

```diff
@@ -33,11 +33,10 @@ const App = () => {
       ) : (
         <div className="w-100 base-font bg-white" lang={locale}>
           <main className="cf w-100 base-font">
-            <Suspense
-              fallback={<ReactPlaceholder showLoadingAnimation={true} rows={30} delay={300} />}
-            >
-              <RouterProvider router={router} />
+            <Suspense fallback={<ReactPlaceholder showLoadingAnimation rows={30} delay={300} />}>
+              <RouterProvider router={router} fallbackElement={<Preloader />} />
             </Suspense>
+            <RouterProvider router={router} fallbackElement={<Preloader />} />
```

Comment:
It looks like there are two instances of the <RouterProvider> component being rendered vertically, which is causing the app to appear twice on the page. This must have been an oversight, and we should remove one of the instances to fix the issue.

Reply:
imagine shipping that haha. thanks for the catch @Aadesh-Baral. appreciate it